boards/arm/mimxrt1024_evk/doc/index.rst

```diff
@@ -0,0 +1,207 @@
+remaining are not used.
+
+Programming and Debugging
+*************************
+
+Build and flash applications as usual (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+Configuring a Debug Probe
+=========================
+
+A debug probe is used for both flashing and debugging the board. This board is
+configured by default to use the :ref:`opensda-daplink-onboard-debug-probe`,
+however the :ref:`pyocd-debug-host-tools` do not yet support programming the
+external flashes on this board so you must reconfigure the board for one of the
+following debug probes instead.
+
+Option 1: :ref:`opensda-jlink-onboard-debug-probe` (Recommended)
+----------------------------------------------------------------
+
+Install the :ref:`jlink-debug-host-tools` and make sure they are in your search
+path.
+
+Follow the instructions in :ref:`opensda-jlink-onboard-debug-probe` to program
+the `OpenSDA J-Link MIMXRT1024-EVK Firmware`_. Check that jumpers J47 and J48
+are **on** (they are on by default when boards ship from the factory) to ensure
+SWD signals are connected to the OpenSDA microcontroller.
+
+Option 2: :ref:`jlink-external-debug-probe`
+-------------------------------------------
+
+Install the :ref:`jlink-debug-host-tools` and make sure they are in your search
+path.
+
+Attach a J-Link 20-pin connector to J55. Check that jumpers J47 and J48 are
+**off** (they are on by default when boards ship from the factory) to ensure
+SWD signals are disconnected from the OpenSDA microcontroller.
+
+Configuring a Console
+=====================
+
+Regardless of your choice in debug probe, we will use the OpenSDA
+microcontroller as a usb-to-serial adapter for the serial console. Check that
+jumpers J50 and J46 are **on** (they are on by default when boards ship from
+the factory) to connect UART signals to the OpenSDA microcontroller.
+
+Connect a USB cable from your PC to J23.
+
+Use the following settings with your serial terminal of choice (minicom, putty,
+etc.):
+
+- Speed: 115200
+- Data: 8 bits
+- Parity: None
+- Stop bits: 1
+
+Flashing
+========
+
+Here is an example for the :ref:`hello_world` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: mimxrt1024_evk
+   :goals: flash
+
+Open a serial terminal, reset the board (press the SW9 button), and you should
+see the following message in the terminal:
+
+.. code-block:: console
+
+   ***** Booting Zephyr OS v2.4.0-rc1 *****
+   Hello World! mimxrt1024_evk
+
+Debugging
+=========
+
+Here is an example for the :ref:`hello_world` application.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: mimxrt1024_evk
+   :goals: debug
+
+Open a serial terminal, step through the application in your debugger, and you
+should see the following message in the terminal:
+
+.. code-block:: console
+
+   ***** Booting Zephyr OS v2.4.0-rc1 *****
+   Hello World! mimxrt1024_evk
+
+
```

Comment:
The documentation check is failing because you haven't included links here

Reply:
Because the document hasn't been uploaded yet, the link to the document is not available for a while, and I will update it immediately after I get the link.

Reply:
Please squash the doc link changes into the commit `board: arm: Add board support for mimxrt1024_evk`